x-pack/plugins/security_solution/public/common/lib/telemetry/events/ai_assistant/index.ts

```diff
@@ -67,3 +81,23 @@ export const assistantQuickPrompt: TelemetryEvent = {
     },
   },
 };
+
+export const assistantSettingToggledEvent: TelemetryEvent = {
+  eventType: TelemetryEventTypes.AssistantSettingToggled,
+  schema: {
+    isEnabledKnowledgeBase: {
+      type: 'boolean',
+      _meta: {
+        description: 'Is knowledge base enabled',
```

Comment:
No change required, but the way this `description` reads in plain text:

```typescript
description: 'Is knowledge base enabled',
```
 aligns with my (subjective) preference for

```
isKnowledgeBaseEnabled
```
over

```
isEnabledKnowledgeBase
```

, because `isKnowledgeBaseEnabled` reads just like the `description`.